includes/media_bynder.utils.inc

```diff
@@ -27,27 +27,48 @@ function media_bynder_rest_client() {
  * @todo add support for type and effect
  */
 function media_bynder_embed_url($fragment_id, $type=false, $effects=false, $name = null){
-    $current_media = variable_get('current_media');
+    if(!$type){
+        $type = "web";
+    }
+    $base_url = variable_get('media_bynder_url', '');
+    if (substr($base_url, -1) != '/') {
+        $base_url .= '/';
+    }
+    $base_url .= 'm/' . $fragment_id . '/' . $type . '-drupal.jpg';
+    $query = [];
 
-    if(isset($current_media)) {
-        if ($current_media['idHash'] == $fragment_id) {
-            return $current_media['thumbnails']['thul'];
+    if($effects){
+        foreach($effects as $effect_key => $effect){
+            switch($effect_key){
+                case 'image_resize':
+                    $query['h'] = $effect['height'];
+                    $query['w'] = $effect['width'];
+                    break;
+            }
         }
     }
-     return FALSE;
+    return $base_url . '?' . http_build_query($query);
 }
 
 /**
  * Parse the style out of the uri.
  */
 function media_bynder_uri_parse_style_url($uri) {
-    $fragment_id = media_bynder_uri_parse_url($uri)['i'];
-    $current_media = variable_get('current_media');
-
-    if(isset($current_media)) {
-        if ($current_media['idHash'] == $fragment_id) {
-            return $current_media['thumbnails']['thul'];
+    $target = file_uri_target($uri);
+    $parts = explode('/', $target);
```

Comment:
@geonetix validateBynderDrupalUrl() (https://github.com/Bynder/drupal-media-bynder/compare/13927_quickfix#diff-046f2c0a9b9e985bc6cf7f2be99fe901R64) has to replace the previous code because it introduces the bug we try to quickfix. So what is the change you'd like to see?